support-frontend/assets/helpers/page/page.js

```diff
@@ -134,11 +132,22 @@ function init<S, A>(
   pageReducer?: CommonState => Reducer<S, A> | null,
   thunk?: boolean = false,
 ): Store<*, *, *> {
-
   try {
+    /**
+     * If ccpaEnabled initialise CCPA CMP as early
+     * as possible so subsequent call to
+     * onIabConsentNotification returns the correct
+     * consentState.
+    * */
+    if (ccpaEnabled()) {
+      initCMP({
+        useCcpa: true,
+      });
+    }
```

Comment:
To update to V4:

```suggestion
    if (ccpaEnabled()) {
      init({
        inInUsa: ccpaEnabled(),
      });
    }
```

Reply:
CMP v4 is still in beta so I would advise against using it until a stable and thoroughly tested version is released.

Reply:
I've opted to release this with v3 for now and upgrade to v4 when we are ready to release TCFv2.